install_config/install/advanced_install.adoc

```diff
@@ -1972,6 +1972,69 @@ Moving from a single master cluster to multiple masters after installation is
 not supported.
 ====
 
+[discrete]
+[[single-master-multi-node-ai]]
+==== Single Master and Multiple Nodes
```

Comment:
@gaurav-nelson It makes me a little hard to identify the differences with `Single Master, Single etcd, and Multiple Nodes`.

In `Single Master and Multiple Nodes`, we just don't have `infra` nodes? (while comparing with `Single Master, Single etcd, and Multiple Nodes`).

Reply:
Yup, you are right. We already have that in our docs. I will remove this section then.